src/app/services/superadmin-dashboard.service.ts

```diff
@@ -8,26 +8,26 @@ import { AllAdminStatusResponses } from '../models/admin';
   providedIn: 'root',
 })
 export class SuperadminDashboardService {
-  constructor(private http: HttpClient) {}
+  constructor(private http: HttpClient) { }
 
   getPendingRequests(): Observable<AllAdminStatusResponses> {
     const response = this.http.get<AllAdminStatusResponses>(
-      environment.apiEndpointauth + '/superadmin/pendingAdminApproval'
+      environment.apiEndpoint + '/superadmin/pendingAdminApproval'
```

Comment:
create a separate variable and use it everywhere. Always remember to follow DRY(Do Not Repeat Yourself)